ui/src/components/Log/index.jsx

```diff
@@ -190,7 +190,7 @@ export default class Log extends Component {
   };
 
   handleLineNumberChange = lineNumber => {
-    this.setState({ lineNumber });
+    this.setState({ lineNumber, follow: null });
```

Comment:
`follow` is a boolean so we should probably just set it to `false` instead of `null`. I'll pull the code down in a bit to test it.

Reply:
@helfi92 I thought about it but ended up with `null` we are `resetting` the value of `follow` and `null` made sense in that context (The default value of `follow` is `null`).

That being said, I don't have any objections setting it to `false`. Do you want me to?

Reply:
I tend to agree with you. We can keep it as `null` :)